builds/e2e/templates/e2e-run.yaml

```diff
@@ -15,7 +15,15 @@ steps:
     }
     else
     {
-      sudo --preserve-env dotnet vstest $testFile --logger:trx
+      # Unfortunately CentOs has some failing test that need to be worked on.
```

Comment:
This works, but we already have a pattern for disabling tests. Search for tests that call `Assert.Ignore("...")`, e.g. in `PriorityQueue.cs`. I'd prefer that you not introduce another pattern.

Reply:
I see what you mean. Though it looks like there is no easy way for a test to detect if is running on CentOs from within the dotnet. I'm thinking to use the category as a work around to disable test until I bring all the test up on CentOs. 
And for the sake of consistency, I could convert the aforementioned PriorityQueue Tests into using Category-blacklist. Would that work for you?

Reply:
Just keep what you have here. Don't convert any of the existing tests.

With `Assert.Ignore`, disabled tests show as "skipped", which is a nice, visible reminder, so I'd like to keep that.

Reply:
I suspect something could be added to `OsPlatform`--we already have methods there like `IsWindows()` and `Is64Bit()` so it seems like a natural fit. But if you need to call `lsb_release` or something similar, then you have to handle async, etc...not worth the effort IMO. The `CentOsSafe` category is temporary, so no biggie.